plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -121,7 +121,7 @@ public DistributedQueryRunner build()
     public static DistributedQueryRunner createDeltaLakeQueryRunner(String catalogName)
             throws Exception
     {
-        return createDeltaLakeQueryRunner(catalogName, ImmutableMap.of(), ImmutableMap.of());
+        return createDeltaLakeQueryRunner(catalogName, ImmutableMap.of(), ImmutableMap.of("delta.register-table-procedure.enabled", "true"));
```

Comment:
I don't think we want to enable this config property in the query runner. Each test should set this explicitly.